src/endpoints/v1.es6.js

```diff
@@ -234,6 +239,21 @@ class APIv1Endpoint {
             return null;
           }
         });
+      },
+
+      popular: function (options = {}) {
```

Comment:
`popular` shouldn't be a top level endpoint - only `get`, `post`, and other HTTP verbs. This should, instead, be built in using `query` in the `get` function.